subsys/usb/device_next/Kconfig

```diff
@@ -47,6 +47,14 @@ config USBD_MSG_SLAB_COUNT
 	help
 	  Maximum number of USB device notification messages that can be queued.
 
+config USBD_MSG_WORK_DELAY
+	int "USB device notification messages work delay"
+	range 1 100
+	default 1
+	help
+	  Message work may need to be delayed because the device stack is not
+	  yet ready to publish the message. The delay unit is microseconds.
```

Comment:
The config is used with `K_MSEC()` that defines the time in milliseconds not microseconds.